### PR TITLE
add logLevel in helm config

### LINKED ISSUE #722: add logLevel in helm config
**Is your feature request related to a problem?/Why is this needed**

**Describe the solution you'd like in detail**

Driver log level is set as `5` by default, while in e2e test, we want to set as `10` to show more logs, need to add `controller.logLevel` and `node.logLevel` in helm config to supprt this:

https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/09466126dce7111dcac8968a9a6abc229a93ae0b/charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-node.yaml#L83
https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/09466126dce7111dcac8968a9a6abc229a93ae0b/charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml#L135

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by boddumanohar**
**What type of PR is this?**
Driver log level is set as 5 by default, while in e2e test, we want to set as 10 to show more logs, need to add controller.logLevel and node.logLevel in helm config to supprt this:

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #722 

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [x] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: charts/latest/azuredisk-csi-driver/values.yaml [hunk 1]
```diff
@@ -68,7 +70,7 @@ windows:
       repository: mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar
       tag: v2.0.1
       pullPolicy: IfNotPresent
-
```
Comment (andyzhangx) [inline charts/latest/azuredisk-csi-driver/values.yaml:71]: remove empty line

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

File: charts/latest/azuredisk-csi-driver/values.yaml (hunk 1 above)
Comment (boddumanohar) [reply charts/latest/azuredisk-csi-driver/values.yaml:71]: its a trailing whitespace that i removed (editor has done it). do you suggest me to remove the line altogether?

**Comment by andyzhangx** [comment]
/hold
hold a while, need to release GA 1.0.0 first, and then continue this PR, thanks.

File: charts/latest/azuredisk-csi-driver/values.yaml (hunk 1 above)
Comment (andyzhangx) [reply charts/latest/azuredisk-csi-driver/values.yaml:71]: no, thanks.

**Comment by boddumanohar** [comment]
@andyzhangx  where are planning to release GA 1.0.0?

**Comment by andyzhangx** [comment]
> /hold
> hold a while, need to release GA 1.0.0 first, and then continue this PR, thanks.

very soon, hold a day or two.

**Comment by andyzhangx** [comment]
would be addressed by https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/728

**Pull request closed without merging**